### PR TITLE
Cleanup system app code

### PR DESCRIPTION
**Pull request by AndreasMoth**
Cleaning up system app code to be more aligned:
- Fix casings
- Sort permissions
- trim whitespaces
- remove empty sections
- remove redundant app areas

Fixes AB#492871

### CODE REVIEW

File: src/System Application/App/Camera and Media Interaction/src/Camera.Page.al [hunk 1]
```diff
@@ -136,12 +136,12 @@ page 1908 Camera
...
     /// <param name="InStream">The InStream to read the picture from.</param>
     /// <error>The picture is not available.</error>
-    procedure GetPicture(InStream: Instream)
+    procedure GetPicture(InStream: InStream)
     begin
         CameraPageImpl.GetPicture(InStream);
     end;
 
-    trigger CameraProvider::PictureAvailable(PictureName: Text; PictureFilePath: Text)
+    trigger CameraProvider::"CameraProvider::PictureAvailable"(PictureName: Text; PictureFilePath: Text)
```
Comment (IhorHandziuk) [inline src/System Application/App/Camera and Media Interaction/src/Camera.Page.al]: This doesn't seem right 🙂

File: src/System Application/App/Data Compression/src/dotnet.al [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.IO;
+
+dotnet
+{
+
+
```
Comment (IhorHandziuk) [inline src/System Application/App/Data Compression/src/dotnet.al]: empty lines

**Review by IhorHandziuk — DISMISSED** [review]

File: src/System Application/App/BLOB Storage/src/PersistentBlobImpl.Codeunit.al [hunk 3]
```diff
@@ -10,7 +10,7 @@ codeunit 4108 "Persistent Blob Impl."
     Access = Internal;
     InherentEntitlements = X;
     InherentPermissions = X;
-    Permissions = TableData "Persistent Blob" = rimd;
+    Permissions = tabledata "Persistent Blob" = rimd;
```
Comment (JesperSchulz) [inline src/System Application/App/BLOB Storage/src/PersistentBlobImpl.Codeunit.al:13]: Makes no sense to me that tabledata would be all lowercase, but I guess that's a choice someone made :-) Should we create a bug for the compiler?

**Review by JesperSchulz — DISMISSED** [review]
Love it. Calms my OCD ;-)

**Review by IhorHandziuk — DISMISSED** [review]

**Review by darjoo — DISMISSED** [review]

File: src/System Application/App/BLOB Storage/src/PersistentBlobImpl.Codeunit.al (hunk 3 above)
Comment (AndreasMoth) [reply src/System Application/App/BLOB Storage/src/PersistentBlobImpl.Codeunit.al:13]: Created bug: [Bug 492872](https://dynamicssmb2.visualstudio.com/Dynamics%20SMB/_workitems/edit/492872): Object references in properties do not use Pascal Casing by default

**Review by JesperSchulz — APPROVED** [review]

**Review by WaelAbuSeada — APPROVED** [review]

**Review by IhorHandziuk — APPROVED** [review]

**Pull request merged**